gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/UpdateApplicationEntity.java

```diff
@@ -76,7 +76,7 @@
     private boolean disableMembershipNotifications;
 
     @JsonProperty("api_key_mode")
-    @Schema(description = "The API key mode used for this application.", allowableValues = { "UNSPECIFIED", "SHARED", "EXCLUSIVE" })
+    @Schema(description = "The API key mode used for this application.")
```

Comment:
I agree, but in that case, shouldn't we add @Schema(enumAsRef = true) in the ApiKeyMode enum ? (This enum has been added with the rebase)